_benchmark/user-guide/concepts.md

```diff
@@ -25,6 +25,93 @@ A workload is a specification of one or more benchmarking scenarios. A workload
 - One or more data streams that are ingested into indexes.
 - A set of queries and operations that are invoked as part of the benchmark.
 
+## Throughput and latency
+
+At the end of each test, OpenSearch Benchmark produces a table that summarizes the following: 
+
+- [Service time](#service-time) 
+- Throughput
+- [Latency](#latency)
+- The error rate for each completed task or OpenSearch operation.
+
+While the definition for _throughput_ remains consistent with other client-server systems, the definitions for `service time` and `latency` differ from most client-server systems in the context of OpenSearch Benchmark. The following table compares the OpenSearch Benchmark definition of service time and latency versus the common definitions for a client-server system.
+
+| Metric | Common definition | **OpenSearch Benchmark definition**	|
+| :--- | :--- |:--- |
+| **Throughput** | The number of operations completed in a given period of time.	| The number of operations completed in a given period of time. |
+| **Service time**	| The amount of time that the server takes to process a request, from the point it receives the request to the point the response is returned. </br></br> It includes the time spent waiting in server-side queues but _excludes_ network latency, load balancer overhead, and deserialization/serialization. | The amount of time that it takes for `opensearch-py` to send a request and receive a response from the OpenSearch cluster. </br> </br> It includes the amount of time that it takes for the server to process a request and also _includes_ network latency, load balancer overhead, and deserialization/serialization.  |
```

Comment:
Correct me if I am wrong @dblock, @IanHoang.

In OpenSearch benchmarks, Service Time is calculated in osbenchmark/client.py using the following trace configuration:


```
trace_config = aiohttp.TraceConfig()
trace_config.on_request_start.append(on_request_start)
trace_config.on_request_end.append(on_request_end)
```
I believe the first definition (common definition) in the above documentation code aligns more accurately with our interpretation of Service Time.

Service Time: Represents the interval from the server receiving the request to the server sending the response.

Additional Information:
I attempted to calculate the times by implementing a function perform_request within the AIOHttpConnection in osbenchmark/async_connection.py. The times obtained indicate that the calculated service time doesn't include client processing time.

```
async def perform_request(self, method, url, params=None, body=None, timeout=None, ignore=(), headers=None):
    print("AIOHttpConnection perform_request start time", time.perf_counter())
    status, headers, raw_data = await super().perform_request(method=method, url=url, params=params, body=body, timeout=timeout, ignore=ignore, headers=self.headers)
    print("AIOHttpConnection perform_request end time", time.perf_counter())
    return status, headers, raw_data
```
Sample output:
```
AIOHttpConnection perform_request start time 19.0016295
AIOHttpConnection perform_request end time 19.007145792
service time start 19.001905542
service time end 19.007082167
service time 0.005176625000000712
AIOHttpConnection perform_request start time 19.008452584
AIOHttpConnection perform_request end time 19.012769334
service time start 19.008625875
service time end 19.012713375
service time 0.004087500000000688
```